opensrp-chw/src/main/assets/thinkmd_assessment_parser.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<html>
```

Comment:
isn't this file in the core Library https://github.com/OpenSRP/opensrp-client-chw-core/pull/390 why is this repeated here

Reply:
This file is removed from the chw-core Library. This is now only present in the chw project.